l1-contracts/deploy-scripts/upgrade/DefaultEcosystemUpgrade.s.sol

```diff
@@ -1531,6 +1569,15 @@ contract DefaultEcosystemUpgrade is Script, DeployL1Script {
                 initCalldata: ""
             });
             return abi.encode(block.chainid, diamondCut);
+        } else if (compareStrings(contractName, "ChainAssetHandler")) {
```

Comment:
please put this is DeployL1 or DeployUtils instead, it should be a part of the standard deployment